src/command/archive/get/file.c

```diff
@@ -39,6 +45,33 @@ archiveGetFile(
     // Check all files in the actual list and return as soon as one is copied
     bool copied = false;
 
+    RelFileNode *filter_list = NULL;
```

Comment:
How often is the archiveGetFile function called? Maybe it would be worth making the filter_list variable static so that it is only filled the first time it is called?